news/11909-pyupgrade

```diff
@@ -0,0 +1,20 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* <news item>
+
+### Docs
+
+* <news item>
+
+### Other
+
+* Dropped pragma fixes from pre-commit checks. (#11909)
+* Added pyupgrade to pre-commit checks. (#11909)
```

Comment:
* Added pyupgrade to pre-commit checks. (#11909). This change affects many
  files. Existing pull requests may need to be updated, rebased, or merged to
  avoid conflicts.

Reply:
```suggestion
* Added pyupgrade to pre-commit checks. This change affects many
  files. Existing pull requests may need to be updated, rebased, or merged to
  address conflicts. (#11909)
```